2.2: Build fails with error NU5125

I attempted to update our build to use a 2.2.100 RTM candidate of the SDK. This includes changes to NuGet which cause the build to fail with:

> C:\b\w\d1c9b84e58362abc\.dotnet\x64\sdk\2.2.100-rtm-009571\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(202,5): error NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead. [C:\b\w\d1c9b84e58362abc\src\Framework\Microsoft.AspNetCore.All\pkg\Microsoft.AspNetCore.All.pkgproj]

Presumably this was introduced in https://github.com/NuGet/NuGet.Client/pull/2450.

@nkolev92 @rrelyea - are there known issues in this? The project in question, [Microsoft.AspNetCore.All.pkgproj](https://github.com/aspnet/AspNetCore/blob/release/2.2/src/Framework/Microsoft.AspNetCore.All/pkg/Microsoft.AspNetCore.All.pkgproj), is not using a hand-written nuspec.

cc @Eilon blocking issue for 2.2 RTM.

--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -9,6 +9,12 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <NeutralLanguage>en-US</NeutralLanguage>
     <PackageLicenseUrl>https://raw.githubusercontent.com/aspnet/AspNetCore/2.0.0/LICENSE.txt</PackageLicenseUrl>
+    <!--
+      Suppress a warning about upcoming deprecation of PackageLicenseUrl. When embedding licenses are supported,
+      replace PackageLicenseUrl with PackageLicenseExpression.
+    -->
+    <NoWarn>$(NoWarn);NU5125</NoWarn>
+
     <PackageIconUrl>https://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
     <PackageProjectUrl>https://asp.net</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>

